### PR TITLE
RUN-5413 browser view iframes

### PR DESCRIPTION
**Pull request by pbaize**
#### Description of Change
Support Iframes within a browser view

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] automated tests are [added](https://testing-dashboard.openfin.co/#/app/tests/5d28f36dcb360141a7dfe956/edit)
- [x] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [x] PR release notes describe the change in a way relevant to app-developers

#### Release Notes

Notes: Experimental BrowserView Support

### CODE REVIEW

File: src/browser/api/window.js [hunk 1]
```diff
@@ -1002,6 +970,7 @@ Window.create = function(id, opts) {
             name
         });
     }
+    WebContents.setIframeHandlers(browserWindow.webContents, winObj, uuid, name);
```
Comment (licui3936) [inline src/browser/api/window.js]: browserWindow.webContents => webContents , it's already defined before.

File: src/shapes.ts [hunk 2]
```diff
@@ -28,7 +28,12 @@ export interface ResourceFetchIdentity extends Identity {
     resourceFetch?: boolean;
 }
 
-export type EntityType = 'window' | 'iframe' | 'external connection' | 'unknown';
+export enum EntityType {
+   WINDOW = 'window',
+   IFRAME = 'iframe',
+   EXTERNAL = 'external connection',
+   UNKNOWN = 'unknown'
+}
```
Comment (licui3936) [inline src/shapes.ts]: Do we need new entity type for browser view?

File: src/shapes.ts [hunk 3]
```diff
@@ -108,13 +113,11 @@ export interface WebOptions {
 }
 export interface OpenFinWindow extends InjectableContext {
     isIframe?: boolean;
-    parentFrameId?: number;
     _options: WindowOptions;
```
Comment (tgoc99) [inline src/shapes.ts]: we sure we dont need parentFramId in runtime?

Comment (pbaize) [reply src/shapes.ts]: Looks like a core thing only

File: src/browser/api/window.js (hunk 1 above)
Comment (pbaize) [reply src/browser/api/window.js]: It's defined > 500 lines above, I feel like browserWindow.webContents is more readable.

File: src/shapes.ts (hunk 2 above)
Comment (pbaize) [reply src/shapes.ts]: Probably do

**Pull request merged**